genesis/engine/solvers/rigid/gjk_decomp.py

```diff
@@ -0,0 +1,3394 @@
+import taichi as ti
+import genesis as gs
+import genesis.utils.geom as gu
+from .support_decomp import Support
+
+EPA_P2_NONCONVEX = 2
```

Comment:
Use an actual Python Enum for this, i.e.:
```python
class EPA_RETURN_CODE(IntEnum):
    P2_NONCONVEX = 2
    P2_FALLBACK3 = 3
    P3_BAD_NORMAL = 4
    P3_INVALID_V4 = 5
    P3_INVALID_V5 = 6
    P3_MISSING_ORIGIN = 7
    P3_ORIGIN_ON_FACE = 8
    P4_MISSING_ORIGIN = 9
    P4_FALLBACK3 = 1
```

By the way, why start at 2 instead of 0? Add a comment to explain what P2, P3, P4 prefixes mean if you know it.